### DISCUSSION TITLE
Icon with unocss

### DISCUSSION THREAD

**Discussion by lyneal** [quasarframework/quasar#16716] [General - Components / Directives / etc]
Is it possible to use icons installed via "unocss" in components that include "icon"?

Example: `<q-btn icon="i-ph-amazon-logo"/>`

**Comment by supebase** [comment]
I'm using the [Remixicon](https://remixicon.com) package, here's the code:

`import { useQuasar } from "quasar";`

`const $q = useQuasar();`

`$q.iconMapFn = (iconName) => {`
  `// Determine whether the icon name starts with "app:"`
  `if (iconName.startsWith("app:") === true) {`
    `// If so, remove the prefix and get the icon name`
    `const name = iconName.substring(3);`
    `// Returns an object containing the icon class name`
    `return {`
      `cls: "ri-" + name,`
    `};`
  `}`
`};`

So, components code like this:

`
<q-btn icon="app:vuejs-line" />
`
